scripts/gatsby-source-iiif/indexSearch.js

```diff
@@ -64,35 +66,11 @@ const determineProvider = (manifest) => {
   return 'Rare Books and Special Collections'
 }
 
-const getNumberWithOrdinal = (n) => {
-  const s = ['th', 'st', 'nd', 'rd']
-  const v = n % 100
-  return n + (s[(v - 20) % 10] || s[v] || s[0])
-}
-
-const getCenturyTags = (dates) => {
-  if (!dates) {
-    return ['undated']
-  }
-  if (Array.isArray(dates)) {
-    dates = dates[0]
-  }
-  dates = dates.toString()
-  const mappedDates = dates.match(/([0-9]{4})/g)
-  if (!mappedDates) {
-    console.error('date not mapped', dates)
-    return ['undated']
-  }
-
-  let years = dates.match(/([0-9]{4})/g).map((year) => {
-    year = Math.ceil(year / 100)
-    return getNumberWithOrdinal(year) + ' Century'
-  })
-  if (years.length === 0) {
-    years = ['undated']
-  }
-
-  return years
+const indexSettings = {
```

Comment:
can you remove these indexSettings?

Reply:
yup i have been wondering about that but did not want to remove it.

Reply:
looks good